avm/res/resources/deployment-script/version.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "0.2",
+  "version": "0.3",
```

Comment:
Hey @segraef, it's ultimately up to @sebassem, but I just got to ask: Why the version.json update? 😄 From the looks of it, it's just a new test that is added, right?

Reply:
This triggered me

![image](https://github.com/Azure/bicep-registry-modules/assets/19261257/083a609e-f751-436a-9ad3-0246d0b1cc6a)

so yes you're right we can keep `minor` and let `patch` increase automatically.